test/functional/ui/decorations_spec.lua

```diff
@@ -866,6 +866,34 @@ end]]
   end)
 end)
 
+describe('extmark conceal', function()
+  local screen, ns
+  before_each( function()
```

Comment:
Don't see a reason for a new describe() group nor this before_each. So I moved the test to the above group.